electronics_lib/JlcTable.py

```diff
@@ -1,10 +1,9 @@
 import re
 from .PartsTable import *
-import numpy as np
-class JLcTable(LazyTable):
-  """Shared base class for JLCPCB product tables that contains common row definitions."""
 
-  DISTRIBUTER_PART_NUMBER = 'LCSC PART'
+class JlcTable(LazyTable):
+  """Shared base class for JlCPCB product tables that contains common row definitions."""
+  DISTRIBUTER_PART_NUMBER = PartsTableColumn(str)
```

Comment:
```suggestion
  JLC_PART_NUMBER = PartsTableColumn(str)
```

This is specific to JLC - the only use for this is to generate JLC pick'n'place data. Unlike manufacturer part number, this (probably) isn't going to have any more general usage, so it's safe to name it JLC-specific.

(and also change other references for consistency)